model.rb

```diff
@@ -41,6 +41,16 @@ def ubid
     @ubid ||= UBID.from_uuidish(id).to_s.downcase
   end
 
+  NON_ARCHIVED_MODELS = ["DeletedRecord", "Semaphore", "Sshable", "StorageKeyEncryptionKey"]
```

Comment:
Odds that we keep this up to date correctly are quite bad. How about detecting encrypted columns? 

```ruby
Sshable.instance_variable_get(:@column_encryption_metadata).keys
```

the `.keys` (and not `&.keys`, or otherwise ignoring nillity) is important.

For extra robustness -- worthwhile perhaps in this case -- write tests that ensure this code detects a few encrypted columns that we know exist for sure.